### DISCUSSION TITLE
Changing URL of Meeting Guide

### DISCUSSION THREAD

**Discussion by DanMTampa** [code4recovery/12-step-meeting-list#1212] [Q&A]
We are changing the location of our meeting guide and I need to know how to make sure the new meeting guide is syncing with the meeting guide app. 

I filled out the form: https://meetingguide.helpdocs.io/article/jsydw3bxw8-connection-form and submitted the info. 

Our meeting guide was at (still active for now) https://guide.ncintergroup.com/Guide/meetings/

The new location is @ https://aadistrict28.org/meetings/

I am not sure what else I need to do to effect this change.

**Comment by kiyote33** [comment]
Hi @DanMTampa, it seems like you've done what's needed to advise GSO of your new feed URl by filling out and submitting the necessary form. If you are having issues or there is some kind of delay, you might try contacting the GSO support team via their Contact form at [https://meetingguide.helpdocs.io/](https://meetingguide.helpdocs.io).   Hope this helps...
